### ISSUE TITLE
Link in Readme.md leads to 404 

### ISSUE THREAD

**Issue by TheRealMephisto** [geerlingguy/drupal-pi#54]
The link in the readme linking to http://www.rackspace.com/knowledge_center/article/how-do-i-modify-my-hosts-file
leads to a 404 Error on rackspace.com
Looks like they have removed this page.

I assume what was explained there is how to modify /etc/hosts file?

**Comment by geerlingguy** [comment]
Yeah, that's correct. I'll have to find another good guide that covers Mac/Windows/Linux to link to instead.
